pkg/mounter/mounter-s3fs.go

```diff
@@ -185,10 +204,33 @@ func (s3fs *S3fsMounter) Unmount(target string) error {
 		if err != nil {
 			return err
 		}
+
+		if pathExist {
+			passwdFile := path.Join(metaPath, passFile)
+			err = os.Remove(passwdFile)
+			if err != nil {
+				klog.Errorf("S3FSMounter Unmount: Cannot remove password file %s: %v", metaPath, err)
+				return fmt.Errorf("S3FSMounter Unmount: Cannot remove password file %s: %v", metaPath, err)
+			}
+		}
+
 		return nil
 	}
 	klog.Info("NodeServer Unmounting...")
-	return s3fs.MounterUtils.FuseUnmount(target)
+	err = s3fs.MounterUtils.FuseUnmount(target)
```

Comment:
Move this logic to `else` section and keep password file deletion logic at one place only at the end.